Package.swift

```diff
@@ -2,9 +2,19 @@ import PackageDescription
 
 let package = Package(
     name: "ReactiveSwift",
-    dependencies: [
-        .Package(url: "https://github.com/antitypical/Result.git", majorVersion: 3, minor: 0)
-    ],
+    dependencies: {
+        #if os(macOS)
+            return [
+                .Package(url: "https://github.com/antitypical/Result.git", majorVersion: 3, minor: 0),
+            ]
+        #else
+            return [
+                .Package(url: "https://github.com/antitypical/Result.git", majorVersion: 3, minor: 0),
+                .Package(url: "https://github.com/Quick/Nimble", majorVersion: 5, minor: 0),
+                .Package(url: "https://github.com/Quick/Quick", majorVersion: 0, minor: 10),
+            ]
+        #endif
```

Comment:
It seems like it'd be easier to only `#if`/`#else` around the added dependencies.

```swift
let package = Package(
    name: "ReactiveSwift",
    dependencies: {
        return [
            .Package(url: "https://github.com/antitypical/Result.git", majorVersion: 3, minor: 0),
        #if !os(macOS)
            .Package(url: "https://github.com/Quick/Nimble", majorVersion: 5, minor: 0),
            .Package(url: "https://github.com/Quick/Quick", majorVersion: 0, minor: 10),
        #endif
        }(),
…
```